presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestVerificationManager.java

```diff
@@ -131,30 +131,23 @@ public void setup()
     }
 
     @Test
-    public void testFailureRequeued()
+    public void testFailureResubmitted()
```

Comment:
Does this test that covers the case that was failing before?  Or if not, is there a test that does?

Reply:
No it does not. Adding that test a bit more complex, and I add a TODO here. Can we proceed with the PR so that we can use it for 0.234 verification? I'll add the test shortly. Thanks!